apps/epic-web/src/pages/workshops/index.tsx

```diff
@@ -116,8 +118,8 @@ const WorkshopsPage: React.FC<{
             {workshops.map((workshop, i) => {
               return (
                 <ModuleProgressProvider moduleSlug={workshop.slug.current}>
-                  <WorkshopTeaser
-                    workshop={workshop}
+                  <Teaser
+                    module={workshop}
                     key={workshop.slug.current}
                     index={i}
```

Comment:
Since the teaser cards could either be Bonus modules or Workshops. I made the component a bit more generic